docker-compose.yml

```diff
@@ -33,6 +33,9 @@ services:
       - AWS_S3_ENDPOINT_URL
       - AWS_S3_REGION_NAME
       - AWS_S3_CUSTOM_DOMAIN
+      - DJANGO_Q_SYNC=True
+      - VAPID_PRIVATE_KEY=/var/app/vapid_private_key.pem
```

Comment:
Please use the base64 value  for this https://openupsa.slack.com/archives/C0189R440L9/p1605781324013200

Reply:
and the public key